.github/workflows/ci.yml

```diff
@@ -47,6 +46,14 @@ jobs:
     - name: Test
       run: |
         cabal test all --enable-tests
+
+    - name: Fetch cabal-docspec                                       
+      run: |                                                          
+        wget https://github.com/phadej/cabal-extras/releases/download/cabal-docspec-0.0.0.20210111/cabal-docspec-0.0.0.20210111.xz &&
+        unxz cabal-docspec-0.0.0.20210111.xz && mv cabal-docspec-0.0.0.20210111 cabal-docspec
+        chmod +x cabal-docspec                                        
+    - name: Run cabal-docspec                                         
+      run: ./cabal-docspec --check-properties --timeout=30
```

Comment:
Is this basically how people are recommended to run `cabal-docspec` locally as well?

Reply:
Pretty much, since it's not on Hackage (cf https://github.com/cdepillabout/password/issues/52#issuecomment-962909061).